security/pfSense-pkg-snort/files/usr/local/www/snort/snort_alerts.php

```diff
@@ -423,62 +412,44 @@ function snort_match_filter_field($flent, $fields, $exact_match = FALSE) {
 	// action lists.
 	switch ($action) {
 		case "action_default":
-			if (isset($alertsid[$gid][$sid])) {
-				unset($alertsid[$gid][$sid]);
+			if (array_get_path($alertsid, "{$gid}/{$sid}")) {
+				array_del_path($alertsid, "{$gid}/{$sid}");
 			}
-			if (isset($dropsid[$gid][$sid])) {
-				unset($dropsid[$gid][$sid]);
+			if (array_get_path($dropsid, "{$gid}/{$sid}")) {
+				array_del_path($dropsid, "{$gid}/{$sid}");
 			}
-			if (isset($rejectsid[$gid][$sid])) {
-				unset($rejectsid[$gid][$sid]);
+			if (array_get_path($rejectsid, "{$gid}/{$sid}")) {
+				array_del_path($rejectsid, "{$gid}/{$sid}");
```

Comment:
You can safely array_del_path() without the conditional, if the path does not exist it does nothing.